### PR TITLE
Dataup 187 move globus upload below drag and drop

### PR DESCRIPTION
**Pull request by eamahanna**
# Description of PR purpose/changes

This PR moves the upload with globus and upload with url buttons bellow the dropzone. It also edits the text in the drop zone.
Screen Shot 2020-10-12 at 3 05 10 PM

# Jira Ticket / Issue #
https://kbase-jira.atlassian.net/browse/DATAUP-187
- [x] Added the Jira Ticket to the title of the PR e.g. (DATAUP-69 Adds a PR template)

# Testing Instructions
* Details for how to test the PR: 
Unit tests: execute `make test-frontend-unit`

To view changes locally:
1. navigate to the narrative repo and execute `kbase-narrative`
2. create a new narrative
3. In the new narrative click the add data button and open the import tab
4. You will see the changes in on the import tab

# Dev Checklist:

- [x] My code follows the guidelines at https://sites.google.com/truss.works/kbasetruss/development
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have run Black and Flake8 on changed Python Code manually or with git precommit (and the travis build passes)

# Updating Version and Release Notes (if applicable)

- [ ] [Version has been bumped](https://semver.org/) for each release
- [ ] [Release notes](/RELEASE_NOTES.md) have been updated for each release (and during the merge of feature branches)

### CODE REVIEW

File: test/unit/spec/narrative_core/upload/stagingAreaViewer-spec.js [hunk 1]
```diff
@@ -99,14 +99,19 @@ define ([
...
                 });
         });
 
         it('Should render properly without a Globus linked account', () => {
-            expect($targetNode.html()).not.toContain('Or upload to this staging area by using');
+            var $globusButton = $targetNode.find('.globus_not_linked');
+            expect($globusButton).toBeDefined();
+            expect($globusButton.html()).toContain('Upload with Globus');
+            expect($globusButton.html()).toContain('https://docs.kbase.us/data/globus');
         });
```
Comment (leia-sefkin) [inline test/unit/spec/narrative_core/upload/stagingAreaViewer-spec.js:115]: Recommend adding a quick test for the Upload with URL button for comprehensive coverage.

**Review by leia-sefkin — COMMENTED** [review]
Feels somewhat disingenuous to review and approve since I wrote a decent chunk of this 😬  but with that caveat it looks good overall to me. Only one small comment about test coverage.

File: test/unit/spec/narrative_core/upload/stagingAreaViewer-spec.js [hunk 2]
```diff
@@ -88,7 +88,7 @@ define ([
...
 
         it('Should render properly with a Globus linked account', (done) => {
-            let $node = $('<div>'),
+            var $node = $('<div>'),
```
Comment (ialarmedalien) [inline test/unit/spec/narrative_core/upload/stagingAreaViewer-spec.js:91]: ? why the change?

File: kbase-extension/static/kbase/js/widgets/narrative_core/upload/stagingAreaViewer.js [hunk 3]
```diff
@@ -166,7 +166,7 @@ define([
 
             // Add ACL before going to the staging area
             // If it fails, it'll just do so silently.
-            var $globusLink = this.$elem.find('.globus_link');
+            var $globusLink = this.$elem.find('a.globus_linked');
```
Comment (ialarmedalien) [inline kbase-extension/static/kbase/js/widgets/narrative_core/upload/stagingAreaViewer.js:169]: I'm not sure if I'm looking at the wrong file, but in `kbase-extension/static/kbase/templates/data_staging/ftp_file_header.html` below, it's a button with the class `globus_linked` and the `a` element has the class `globus_link`. What am I missing?

Comment (leia-sefkin) [reply kbase-extension/static/kbase/js/widgets/narrative_core/upload/stagingAreaViewer.js:169]: Ah good catch: I don't think we need the globus_link class on the <a> element in ftp_file_header - just an artifact at this point. 

For context - the original class name was "globus_link" but I found it somewhat confusing since it referred to two separate links (one where the user was connected/linked to globus and one where not), hence the rename.

File: test/unit/spec/narrative_core/upload/stagingAreaViewer-spec.js (hunk 2 above)
Comment (eamahanna) [reply test/unit/spec/narrative_core/upload/stagingAreaViewer-spec.js:91]: Generally I try and avoid using `let` because it can cause unanticipated bugs at times because it's scope acts pretty differently than `var`.

File: kbase-extension/static/kbase/js/widgets/narrative_core/upload/stagingAreaViewer.js (hunk 3 above)
Comment (eamahanna) [reply kbase-extension/static/kbase/js/widgets/narrative_core/upload/stagingAreaViewer.js:169]: fixed!

**Review by leia-sefkin — APPROVED** [review]
LGTM

**Review by bio-boris — APPROVED** [review]
